stitch/stitch.py

```diff
@@ -125,17 +142,25 @@ def __init__(self, name, to='html',
             output format
         standalone : bool, default True
             whether to make a standalone document
+        self_contained: bool, default True
         warning : bool, default True
             whether to include warnings (stderr) in the ouput.
         error : ``{"continue", "raise"}``
             how to handle errors in the code being executed.
+        prompt : str, default None
+        use_prompt : bool, default False
+            Whether to use prompt prefixes in code chunks
+        extra_args : list of str, default None
```

Comment:
How about calling this `pandoc_args`, or `extra_pandoc_args`?

Reply:
May be `pandoc_extra_args` would be OK. Please note that now these args are only useable via `ast-filter` (only it's `Stitch` constructor gets `extra_args`).

About `ensure that extra_args are passed through correctly`:

Only thing we can do is to check if it's a lists of stings. I not it would lead to pypandoc/pandoc error. And even if it's a list of stings it also can lead to pandoc error (for example if there's a wrong option).

So I see no point in checking `pandoc_extra_args` at all.

Reply:
About `ensure that extra_args are passed through correctly`: do you mean it would be tests that ensure that future commints would not break this functionality?

It's not clear to me what should do the test to `ensure that extra_args are passed through correctly`...

Maybe create Stitch object with `pandoc_extra_args` (`stitcher = Stitch(pandoc_extra_args=args)`) and then check if `stitcher.pandoc_extra_args == args`?

But that looks pointless...